common/reactive/src/test/java/io/helidon/common/reactive/SequentialSubscriberTest.java

```diff
@@ -98,7 +99,7 @@ public void onComplete() {
         } catch (InterruptedException e) {
             executorService.shutdownNow();
         }
-        assertFalse(errorFound.isPresent(), () -> errorFound.get());
+        assertThat(errorFound.orElse(""), errorFound.isPresent(), is(false));
```

Comment:
I can use [specific mathed](https://github.com/helidon-io/helidon/pull/4908#discussion_r973594879) , but I don't find it :( . Can you show?

Reply:
See https://github.com/helidon-io/helidon/blob/2ad014cc3bfbe581c82ef3ac15690cc555f44b16/config/config/src/test/java/io/helidon/config/FileConfigSourceTest.java#L97